src/sdkToEventsApiConverter.ts

```diff
@@ -56,10 +56,25 @@ export function convertEvents(
         currentConsentState = user.getConsentState();
     }
 
+    // determine what timestamp, if any, to use for the batch
+    const { omitBatchTimestamp } = mpInstance._Store.SDKConfig;
+    const { batchTimestampUnixtimeMsOverride } = mpInstance._Store;
```

Comment:
I just noticed that you're puling this from `_Store` as we discussed, but I don't see any code path where `batchTimestampUnixtimeMsOverride` is being added to the actual `_Store`. If I remember correctly, we discussed that the override would be added as an `eventOption`.

I think within `APIClient.sendEventToServer`, you can pass in `batchTimestampUnixtimeMsOverride` through `_options` and use that to set it in the `_Store`, where it can then be picked up here.

Reply:
Reminder that it should be added to each of `logPageView`, `logEvent`, and `logProductAction`.